src/main/java/com/joy/joyfillandroid/TextField.kt

```diff
@@ -0,0 +1,175 @@
+package com.joy.joyfillandroid
+
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Color
+import android.graphics.Typeface
+import android.os.Build
+import android.view.Gravity
+import android.widget.LinearLayout
+import androidx.appcompat.widget.AppCompatTextView
+
+class TextField(context: Context?) : AppCompatTextView(context!!) {
+
+    private var borderColor: Int = 0
+    private var cornerRadius: Int = 0
+    private var borderWidth: Int = 0
+    private var isBold: Boolean = false
+    private var isItalic: Boolean = false
+    private var fontName: String? = null
+
+    companion object {
+        private const val DEFAULT_TEXT_SIZE_SP = 6f
+        private const val DEFAULT_TEXT_COLOR = android.R.color.black
+    }
+
+    init {
+        // Read attributes from XML
+
+        text = "Text Field"
```

Comment:
Text Field will this be dynamic?. Also would the "Read attributes from XML" comment be correct? cause you aren't reading here you are assigning values it appears.

Reply:
For now it is static , but it will be dynamic and then user can set the label according to their  need .